staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -71,6 +73,8 @@ type AvailableConditionController struct {
 	syncFn func(key string) error
 
 	queue workqueue.RateLimitingInterface
+	cache map[string]map[string][]string
```

Comment:
needs a comment documenting each key/value meaning, e.g.

// map of service-namespace -> service-name -> apiservice names